be/src/vec/exprs/vexpr.cpp

```diff
@@ -174,218 +174,220 @@ Status VExpr::create_expr(doris::ObjectPool* pool, const doris::TExprNode& texpr
     default:
         return Status::InternalError("Unknown expr node type: {}", texpr_node.node_type);
     }
-    return Status::OK();
-}
+        return Status::OK();
+    }
 
-Status VExpr::create_tree_from_thrift(doris::ObjectPool* pool,
```

Comment:
Why modify here?

Reply:
when I use clang-format.sh  , it make these change...